File: PartyPlanner.Core/Dtos/Item.cs
using MongoDB.Bson.Serialization.Attributes;
using PartyPlanner.Core.Dtos.Interfaces;

namespace PartyPlanner.Core.Dtos
{
    [BsonIgnoreExtraElements]
    public class Item
    {
        public int ItemId { get; set; }
    
        public string Name { get; set; }
        public string Description { get; set; }
        public float Price { get; set; }
        public int Quantity { get; set; }
    }   
}

File: partyplanner.web/src/components/category/item/Item.js
const Item = ({ item, onItemDelete }) => {
    return (
        <div className='subitem'>
            <p>Name: {item.name}</p>
            <p>Price: {item.price}</p>
            <p>Quantity: {item.quantity}</p>
            <button onClick={onItemDelete}>X</button>
        </div>
    );
};

export default Item;